src/styles/Blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+  &__box {
+    display: flex;
+    height: 60px;
+    flex-grow: 1;
+
+    &-logo {
+      position: absolute;
+      z-index: 1;
+      top: 10px;
+      left: 50px;
+      width: 40px;
+      height: 40px;
+    }
+  }
+
+  &__container {
+    display: flex;
+    height: 60px;
+    justify-content: flex-end;
+    margin-left: 0;
+    margin-right: 50px;
+
+    &-titles {
+      list-style: none;
+      align-items: center;
+      display: flex;
+    }
+  }
+}
+
+.menu {
+  color: #060b35;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+  transition: all 300ms ease;
+
+  &:hover {
+    color: #00acdc;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  display: inline-block;
+  margin-left: 0;
+  height: 60px;
+  line-height: 60px;
+
+  &::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background-color: #00acdc;
+    transition: background-color 0.3s;
+    border-radius: 10px;
+    border: 10px solid;
```

Comment:
The border property is used without specifying a color. Consider specifying a color to avoid unexpected results.